### DISCUSSION TITLE
无法使用SuperMap iClient3D 11i(2023) SP1 for Cesium

### DISCUSSION THREAD

**Discussion by wensenhh** [zouyaoji/vue-cesium#515] [Q&A]
![image](https://github.com/zouyaoji/vue-cesium/assets/57422874/875c3119-65f1-4265-82d6-a6c8b85117c0)
cesiumPath 为本地  超图最新的Cesium包
初始化时控制台报错
![image](https://github.com/zouyaoji/vue-cesium/assets/57422874/39087d51-2b00-4400-a0c3-6eb3e57e93f3)
是因为vue-cesium不支持超图吗？还是我应该如何进行配置

**Comment by wensenhh** [comment]
使用官方cesium时是没有问题的，但项目要求必须使用超图的包，更换后报错！ 根据vue-cesium官网例子，
设置cesiumPath: 'https://www.supermapol.com/earth/Build/Cesium/Cesium.js'
可正常渲染天地图，但控制台提示
![image](https://github.com/zouyaoji/vue-cesium/assets/57422874/4a63eb22-effb-4901-b3e3-dcff4635f529)
看来还是跟版本有关系，vue-cesium是否支持最新的cesium，最高能支持到哪个版本

**Reply by zouyaoji** [reply]
警告可以忽略。第一个截图看超图最新的包发出来看看。

**Reply by wensenhh** [reply]
[SuperMap iClient3D 11i(2023) SP1 for Cesium](http://support.supermap.com.cn/DownloadCenter/DownloadPage.aspx?id=2513)

**Reply by zouyaoji** [reply]
我抽时间试试 有结果了给你说

**Reply by zouyaoji** [reply]
滴滴 方便上传到这儿不，没百度网盘会员啊

**Reply by zouyaoji** [reply]
我跟了一下，超图这个版本把Cesium.VERSION都删除了，导致判断Cesium版本失败，这就不好搞了。下个版本先忽略这个错误吧，有问题再说。
